test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require './utils'
+
+def fetch_albums
+  File.readlines('discography.txt').map(&:chomp)
```

Comment:
You could also use [strip](https://apidock.com/ruby/String/strip)  to remove all the blank characters at the start and end of the line